src/docs/knowledge-base/cmsis-packs.md

```diff
@@ -111,7 +111,7 @@ flash_algorithms:
         - size: number
           # The address from on which the new sector size is applicable.
           address: number
-# The core type. Currently valid are [M4, M3, M33, M0, RISCV]
+# The core type. Currently valid are [M4, M3, M33, M0, RISCV, Xtensa]
 core: string
```

Comment:
Not entirely sure if this is correct or the remnant of some past state. It looks like there's no topmost level `core` in the target yml currently, but `cores` as part of elements in `variants`. The valid options also don't line up with the target yml files.

Reply:
Aw shoot, I did not even check that ... Can we merge this as is and I fix that in a separate PR? :)

Reply:
I don't think I should tell you what you can or can't do here 😅 I'm not familiar with most of the stuff in this PR, just found this one spot. All else is up to you :)

Reply:
Well I guess getting this in as is (and unblocking other work) may be more important than an example in the docs. I just don't have approval rights 🙃